### PR TITLE
Se4 35 create running stats page

### PR DESCRIPTION
**Pull request by josegacosta33**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -96,7 +96,13 @@
 			<artifactId>jackson-databind</artifactId>
 			<version>2.12.3</version>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
```
Comment (davidwmathews) [inline pom.xml]: Is this library used anywhere in the code or can it be deleted?

Comment (josegacosta33) [reply pom.xml]: I dont remember editing the pom file at all. I thought that was there before but I can delete.

File: src/main/resources/static/js/home.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 const birthdate = new Date(userBirthdate);
-
```
Comment (kristie-a) [inline src/main/resources/static/js/home.js]: I put the line breaks in my JavaScript code for the home page to make it easier to read. Could you please revert this file back to the original?

Comment (josegacosta33) [reply src/main/resources/static/js/home.js]: Ok I'll add a line break now for home.js

File: src/main/resources/static/js/home.js [hunk 3]
```diff
@@ -36,7 +36,6 @@ function formatBirthday() {
 
     document.getElementById("birthday").innerHTML = birthday + ordinalSuffix;
 }
-
 function convertHeightToCm() {
```
Comment (kristie-a) [inline src/main/resources/static/js/home.js]: Could you please put the line break back before the function here as well? Thank you!

Comment (josegacosta33) [reply src/main/resources/static/js/home.js]: ok

**Review by kristie-a — APPROVED** [review]

**Review by alexDev-92 — APPROVED** [review]

**Pull request merged**